graphics/tiny-renderer/Renderer/obj/x86/Debug/Renderer.csproj.CoreCompileInputs.cache

```diff
@@ -0,0 +1 @@
+9bb39d8edca43d4168e9cdeb34c44db91c68d5c1
```

Comment:
@conceptdev I don't think you meant to add this file. Looks like we need to update our .gitignore.

Reply:
Well now I'm confused because there are a ton of populated `obj` directories in tiny-renderer. So, your call!

Reply:
wth - no, that was unintentional... :facepalm: